### ISSUE TITLE
Error when creating a new user

### ISSUE THREAD

**Issue by jmontoyaa** [claroline/Claroline#705]
URL: Claroline/web/app_dev.php/register/form

I fill the form and then I have this error:
```
[1] Symfony\Component\Debug\Exception\ContextErrorException: Warning: Missing argument 5 for Claroline\CoreBundle\Manager\UserManager::createUser(), called in /var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Manager/UserManager.php on line 265 and defined
    at n/a
        in /var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Manager/UserManager.php line 127

    at Symfony\Component\Debug\ErrorHandler->handleError('2', 'Missing argument 5 for Claroline\CoreBundle\Manager\UserManager::createUser(), called in /var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Manager/UserManager.php on line 265 and defined', '/var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Manager/UserManager.php', '127', array('user' => object(User), 'sendMail' => true, 'additionnalRoles' => array(object(Role)), 'model' => null))
        in /var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Manager/UserManager.php line 127

    at Claroline\CoreBundle\Manager\UserManager->createUser(object(User), true, array(object(Role)))
        in /var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Manager/UserManager.php line 265

    at Claroline\CoreBundle\Manager\UserManager->createUserWithRole(object(User), 'ROLE_USER')
        in /var/www/html/Claroline/vendor/claroline/core-bundle/Claroline/CoreBundle/Controller/RegistrationController.php line 135

    at Claroline\CoreBundle\Controller\RegistrationController->registerUserAction()
        in  line 

    at call_user_func_array(array(object(RegistrationController), 'registerUserAction'), array())
        in /var/www/html/Claroline/app/bootstrap.php.cache line 3109

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), '1')
        in /var/www/html/Claroline/app/bootstrap.php.cache line 3071

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), '1', true)
        in /var/www/html/Claroline/app/bootstrap.php.cache line 3222

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Request), '1', true)
        in /var/www/html/Claroline/app/bootstrap.php.cache line 2444

    at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
        in /var/www/html/Claroline/web/app_dev.php line 28

```

**Comment by ngodfraind** [comment]
Thanks for the reports. I'll try to reproduce and fix all of them, then I'll release a new version.

**Comment by ngodfraind** [comment]
Bug introduced by Claroline/CoreBundle@f66fbfa6b24cf7ae2a7ff0c60c2a345b9bacdab4
and fixed by Claroline/CoreBundle@610d1e8b85c149b4bee6df2eea5a579ebaa470a7

The next release will come this evening/afternoon.
